### PR TITLE
Add missing command in /h poll

### PR DESCRIPTION
**Pull request by Distrib-ps**
Hey ! PR for https://www.smogon.com/forums/threads/missing-command-from-h-poll.3674791/

### CODE REVIEW

**Comment by AnnikaCodes** [comment]
This help is really long; maybe we should tuck it into a readmore?

**Comment by Distrib-ps** [comment]
Yes, I think it will be better.

File: server/chat-plugins/poll.ts [hunk 1]
```diff
@@ -537,22 +537,27 @@ export const commands: ChatCommands = {
...
+			`<p><code>/poll htmlcreate(multi) [question], [option1], [option2], [...] </code>- Creates a poll, with HTML allowed in the question and options. Requires: # &<br />` +
+			`<p><code>/poll vote [number]</code> - Votes for option [number].<br />` +
+			`<p><code>/poll timer [minutes]</code> - Sets the poll to automatically end after [minutes]. Requires: % @ # &.<br />` +
+			`<p><code>/poll results </code>- Shows the results of the poll without voting. NOTE: you can't go back and vote after using this.<br />` +
+			`<p><code>/poll display</code> - Displays the poll.<br />` +
+			`<p><code>/poll end </code>- Ends a poll and displays the results. Requires: % @ # &.<br />` +
+			`<p><code>/poll queue [question], [option1], [option2], [...]</code> - Add a poll in queue. Requires: % @ # &<br />` +
+			`<p><code>/poll deletequeue [number]</code> - Deletes poll at the corresponding queue slot (1 = next, 2 = the one after that, etc).<br />` +
+			`<p><code>/poll clearqueue </code>- Deletes the queue of polls. Requires: % @ # &.<br />` +
```
Comment (sirKiraUzumaki) [inline server/chat-plugins/poll.ts:556]:
```suggestion
			`<p><code>/poll htmlcreate(multi) [question], [option1], [option2], [...]</code> - Creates a poll, with HTML allowed in the question and options. Requires: # &<br />` +
			`<p><code>/poll vote [number]</code> - Votes for option [number].<br />` +
			`<p><code>/poll timer [minutes]</code> - Sets the poll to automatically end after [minutes]. Requires: % @ # &.<br />` +
			`<p><code>/poll results</code> - Shows the results of the poll without voting. NOTE: you can't go back and vote after using this.<br />` +
			`<p><code>/poll display</code> - Displays the poll.<br />` +
			`<p><code>/poll end</code> - Ends a poll and displays the results. Requires: % @ # &.<br />` +
			`<p><code>/poll queue [question], [option1], [option2], [...]</code> - Add a poll in queue. Requires: % @ # &<br />` +
			`<p><code>/poll deletequeue [number]</code> - Deletes poll at the corresponding queue slot (1 = next, 2 = the one after that, etc).<br />` +
			`<p><code>/poll clearqueue</code> - Deletes the queue of polls. Requires: % @ # &.<br />` +
```
Just a small suggestion to make it look more formal and neat.

**Review by sirKiraUzumaki — CHANGES_REQUESTED** [review]

File: server/chat-plugins/poll.ts [hunk 2]
```diff
@@ -537,22 +537,27 @@ export const commands: ChatCommands = {
...
+			`<p><code>/poll timer [minutes]</code> - Sets the poll to automatically end after [minutes]. Requires: % @ # &.<br />` +
+			`<p><code>/poll results</code> - Shows the results of the poll without voting. NOTE: you can't go back and vote after using this.<br />` +
+			`<p><code>/poll display</code> - Displays the poll.<br />` +
+			`<p><code>/poll end</code> - Ends a poll and displays the results. Requires: % @ # &.<br />` +
+			`<p><code>/poll queue [question], [option1], [option2], [...]</code> - Add a poll in queue. Requires: % @ # &<br />` +
+			`<p><code>/poll deletequeue [number]</code> - Deletes poll at the corresponding queue slot (1 = next, 2 = the one after that, etc).<br />` +
+			`<p><code>/poll clearqueue</code> - Deletes the queue of polls. Requires: % @ # &.<br />` +
+			`<p><code>/poll viewqueue</code> - View the queue of polls in the room. Requires: % @ # &<br />` +
+			`</details>`
+		);
```
Comment (AnnikaCodes) [inline server/chat-plugins/poll.ts:559]: You should remove the `<p>` tags here.

**Review by AnnikaCodes — APPROVED** [review]

**Comment by Zarel** [comment]
I think it's important for commands not to be useless without clicking "read more" – Design Standard 3: https://github.com/smogon/pokemon-showdown/blob/master/CONTRIBUTING.md#design-standards

It makes sense to bring `/poll create` above the fold.

**Comment by AnnikaCodes** [comment]
If we're doing that then `/h daily` and `/h dc` should also be changed

**Comment by Zarel** [comment]
c13a9c84fbca11a2 may be good to look for reference, if you want to change `/h daily` and `/h dc`

**Pull request merged**